app/src/main/res/layout/scroll_view.xml

```diff
@@ -0,0 +1,336 @@
+
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guidelinePath2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_percent="0.1" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guidelineHorizontalCenter2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_percent="0.4" />
+
+
+    <ImageView
+        android:id="@+id/imageViewPath2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:contentDescription="@string/todo"
+        app:layout_constraintBottom_toTopOf="@+id/guidelinePath2"
+        app:layout_constraintEnd_toStartOf="@+id/TextViewPath2"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/guidelinePath2"
+        app:srcCompat="@drawable/path" />
+
+    <TextView
+        android:id="@+id/TextViewPath2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="8dp"
+        android:fontFamily="@font/roboto_condensed_light"
+        android:text="@string/path_back"
+        android:textColor="@color/colorPath"
+        app:layout_constraintBottom_toTopOf="@+id/guidelinePath2"
+        app:layout_constraintStart_toEndOf="@+id/imageViewPath2"
+        app:layout_constraintTop_toTopOf="@+id/guidelinePath2"
+        tools:layout_editor_absoluteX="32dp" />
+
+
+    <TextView
+        android:id="@+id/TextViewTegLine2"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:text="@string/textViewTegLine"
+        android:textAppearance="@style/tag"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/TextViewName" />
+
+    <TextView
+        android:id="@+id/TextViewName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_margin="16dp"
+        android:fontFamily="@font/roboto_condensed_light"
+        android:gravity="center_vertical"
+        android:lineHeight="40sp"
+        android:lineSpacingExtra="0sp"
+        android:minLines="2"
+        android:text="@string/MovieDetailsTextViewName"
+        android:textAppearance="@style/name"
+        android:textColor="@color/colorName"
+        app:layout_constraintBottom_toTopOf="@+id/guidelineHorizontalCenter2"
+        app:layout_constraintTop_toTopOf="@+id/guidelineHorizontalCenter2"
+        tools:layout_editor_absoluteX="53dp"
```

Comment:
@kamar19 Please avoid absolute values in layouts and all other not needed attributes.